src/applications/claims-status/containers/OverviewPage.jsx

```diff
@@ -6,16 +6,12 @@ import scrollToTop from '@department-of-veterans-affairs/platform-utilities/scro
 
 import { clearNotification } from '../actions';
 import ClaimComplete from '../components/ClaimComplete';
-// START lighthouse_migration
-import ClaimDetailLayoutEVSS from '../components/evss/ClaimDetailLayout';
```

Comment:
Will we eventually delete this file `../components/evss/ClaimDetailLayout` ? Could we delete it here?

Reply:
No, there are other pages that rely on it so I can't get rid of it until those have been taken care of